buildtest/settings/README.md

```diff
@@ -0,0 +1,47 @@
+# Settings
```

Comment:
Thanks for documenting this for now its okay, we would eventually want this under buildtest docs under Configuring Buildtest https://buildtest.readthedocs.io/en/devel/configuring_buildtest.html

Reply:
Sure, that works. Right now for development I'm putting it where the developer might expect to find it.